src/screens/ENSConfirmRegisterSheet.tsx

```diff
@@ -357,6 +373,12 @@ export default function ENSConfirmRegisterSheet() {
     }, [])
   );
 
+  useEffect(() => {
+    if (step === REGISTRATION_STEPS.REGISTER) {
+      setSendReverseRecord(!isEmpty(changedRecords));
```

Comment:
i think this one is missing the case were the user doesn't have an ens name as primary name and there are not records set, we need to switch turned on but this line would turn it off with no records

Reply:
> When creating ENS profiles, and no records are set, show primary name switch in review screen and set it OFF as default

Isn't this what we want?

Reply:
the difference is `doesn't have an ens name as primary name`, if there's no primary name at all the switch should be ON by default for all cases

Reply:
added a comment here https://linear.app/rainbow/issue/TEAM2-9#comment-78221d81 WDYT?

Reply:
It is on if there is no primary name though?

Reply:
Wait, got my flows mixed up, will fix

Reply:
Wait, got my flows mixed up, will fix

Reply:
fixed in latest commit